src/tvp/treeviews.ts

```diff
@@ -269,7 +269,7 @@ export class TreeViewsManager implements Disposable {
       const windows = await tabpage.windows
       const mbWindow = windows.find(window => allTreeViews.find(wId => window.id === wId) === undefined)
       if (mbWindow === undefined) {
-        const initWidth = this.config.get<number>("initialWidth")
+        const initWidth = this.config.get<number>("initialWidth") ?? 0
```

Comment:
I hope `0` is a sensible default here.

Reply:
Can we default this to what we have in the config, which is 40? If 0 actually would ever be displayed, we'd just see nothing which probably isn't ideal.

Reply:
I've changed this to `!` since it has a default in config, so we can trust the value is there